Assets/Scripts/SimpleTwistControl.lua

```diff
@@ -0,0 +1,44 @@
+local SimpleTwistControl = {
+    Properties = {}
+}
+
+function SimpleTwistControl:OnActivate()
+	self._gravity = -9.81
```

Comment:
Looks like inconsistency with indentations.

Reply:
Indeed, still learning LUA editor (inserts tabs instead of spaces). Ugh.